frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -39,6 +44,15 @@ export const notebookLogic = kea<notebookLogicType>([
         registerNodeLogic: (nodeLogic: notebookNodeLogicType) => ({ nodeLogic }),
         unregisterNodeLogic: (nodeLogic: notebookNodeLogicType) => ({ nodeLogic }),
         setEditable: (editable: boolean) => ({ editable }),
+        insertAfterLastNodeOfType: (nodeType: string, content: JSONContent, desiredInsertPosition) => ({
+            content,
+            nodeType,
+            desiredInsertPosition,
+        }),
+        insertReplayCommentByTimestamp: (timestamp: number, sessionRecordingId: string) => ({
```

Comment:
In theory we wanted to keep this as generic as possible but I think for now this works. Lets have some node-specific methods and maybe that is the sensible approach or maybe we end up realising we should abstract it differently.

Reply:
Yep 👍 easier to abstract from concrete examples